qiskit/pulse/builder.py

```diff
@@ -103,7 +103,7 @@
    d2 = pulse.DriveChannel(2)
 
    with pulse.build(backend) as bell_prep:
-       pulse.u2(0, math.pi, 0)
+       pulse.u3(1.57, 0, math.pi, 0)
```

Comment:
`FakeOpenPulse3Q` doesn't support `u2` gate and this results in the error in the new logic because `target` doesn't include `u2` gate info.